helpers/getFirstParagraph.ts

```diff
@@ -10,7 +10,7 @@ const getFirstParagraph = (content: any) => {
       element.type === "p" && element.children[0].type === "text"
   );
 
-  const text = firstParagraph ? firstParagraph.children[0].text : null;
+  const text = firstParagraph ? firstParagraph.children[0].text : "";
```

Comment:
Why empty string instead of nothing (`null`)?

Reply:
![image](https://github.com/1alexvash/AlexCodeBlog/assets/60061661/74388a6a-0c44-4d6c-b2a5-67969e1d4507)
Because if we return `null`, we'll need to make not-null check, that's why I prefer to return empty string instead of null